src/app/ApplicationTemplate.UWP/Views/Content/Welcome/WelcomePage.xaml

```diff
@@ -2,38 +2,41 @@
 	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
 	  xmlns:local="using:ApplicationTemplate"
-	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
 	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	  xmlns:uc="using:Nventive.View.Converters"
 	  xmlns:u="using:Nventive.View.Controls"
-	  mc:Ignorable="d">
+	  mc:Ignorable="">
 
 	<Grid Background="{StaticResource MaterialOnBackgroundBrush}">
```

Comment:
I would have expected this to be updated too. It doesn't make sense to have a foreground brush set to the Background property.

Reply:
Yeah, but the design for this page is dark blue background and white text which is the opposite of the material nomenclature.

Reply:
The reason we were using ColorXXBrush before was exactly to avoid those type of issue where names that implied a certain usage would end up being use for something else.

Reply:
So like this?
`
<SolidColorBrush x:Key="AlternateBackground"
					Color="{ThemeResource MaterialOnBackgroundColor}" />
<SolidColorBrush x:Key="OnAlternateBackground"
					Color="{ThemeResource MaterialBackgroundColor}" />
`

Reply:
If there are more pages with the same background you could consider changing the resource name to `DarkPageBackground` or something and reuse it

Reply:
I like the `alternateBackground` more cause `DarkPageBackground ` will be light in dark mode.

Reply:
Will be fixed in another PR.